net/net.go

```diff
@@ -16,7 +16,72 @@ limitations under the License.
 
 package net
 
-import "net"
+import (
+	"fmt"
+	"net"
+)
+
+// ParseCIDRs parses a list of cidrs and return error if any is in valid.
+// order is maintained
+func ParseCIDRs(cidrsString []string) ([]*net.IPNet, error) {
+	cidrs := make([]*net.IPNet, 0)
+	for idx, current := range cidrsString {
```

Comment:
nit: can `idx` just be `i`, I feel that `i` is pretty well known for index (in Go at least).

Reply:
nit: current -> cidrString (or cidrStr)